src/server/api/campaign.js

```diff
@@ -235,6 +235,11 @@ export const resolvers = {
         return currentEditors(r.redis, campaign, user)
       }
       return ''
-    }
+    },
+    creator: async (campaign, _, { loaders }) => (
+      campaign.creator_id
```

Comment:
this is fine for now, but can you just add a one line comment that says:
`TODO: optimization to include these fields with organizations.campaigns query`
If you want to stub a bit of that out now it would
1. be `campaign.creator || (campaign.creator_id && loaders.user.load(campaign.creator_id)) || null` here
2. if server/api/organization.js in campaigns you'd also join on user and get creator name.

but the TODO line is fine for now.